### DISCUSSION TITLE
Can nvim-tree.git.enable = true breaks colorscheme ?

### DISCUSSION THREAD

**Discussion by tony-sol** [nvim-tree/nvim-tree.lua#2425] [Q&A] [answered]
I'm not sure, it is a bug or something, but i have a problem: 
every time i open a directory (like `nvim .`), which is a git repository, my colorscheme breaks.

I've tried to enable log, but didn't see any helpful.

Here are my [configs](https://github.com/Tony-Sol/.config/blob/master/nvim/lua/plugins/nvim-tree.lua) and some screencast

https://github.com/nvim-tree/nvim-tree.lua/assets/33671815/2e22a1e6-9c06-4177-aeeb-fac0a468e47e

:

**Comment by gegoune** [comment]
It's impossible to deduct what's going on here. If you believe it's a bug please open bug report following outlined steps.

**Reply by tony-sol** [reply]
I’m not sure is it a bug or just my misconfiguration :c

**Comment by tony-sol** [comment, marked as answer]
Well, i found the problem, which is not in nvim-tree, but in auto-dark-mode.nvim.

But(!) - it's still interesting how connected disabling git intergation in nvim-tree and disableing auto-dark-mode itself.
I think it's all about concurrent loading via lazy.nvim, but not 100% sure

**Comment by alex-courtis** [comment]
I like your lualine setup.
